### PR TITLE
[TRANSLATION] Improvements to portuguese (pt-PT) translation

### PR DESCRIPTION
**Pull request by zecarlos1957**
[BOOTDATA] [CPL] [SHELLEXT] [WIN32] [SHELL32] Improve portuguese (pt-PT) translation

### CODE REVIEW

File: dll/win32/userenv/lang/pt-PT.rc [hunk 1]
```diff
@@ -7,7 +7,7 @@ BEGIN
     IDS_APPDATA "Dados de Programas"
     IDS_DESKTOP "Ambiente de trabalho"
     IDS_FAVORITES "Favoritos"
-    IDS_MYDOCUMENTS "Meus Documentos"
+    IDS_MYDOCUMENTS "Os Meus Documentos"
```
Comment (HBelusca) [inline dll/win32/userenv/lang/pt-PT.rc:10]: Can you also adjust the one in the pt-PT translations in shell32?
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=Meus+Document
the `IDS_PERSONAL` : "OS Meus Documentos" <--- OS in all capitals or just written as "Os" ?

Comment (zecarlos1957) [reply dll/win32/userenv/lang/pt-PT.rc:10]: All other references to this folder in Win32 have the correct name except this one.

Comment (zecarlos1957) [reply dll/win32/userenv/lang/pt-PT.rc:10]: The same problem persists but  in pt-BR

Comment (zecarlos1957) [reply dll/win32/userenv/lang/pt-PT.rc:10]: I see. Thanks

**Pull request merged**